infra/build_deploy_config.py

```diff
@@ -11,10 +11,25 @@
 infra_root = os.path.abspath(os.path.dirname(__file__))
 
 
+
+
 parser = argparse.ArgumentParser(description=__doc__)
 parser.add_argument("component")
 args = parser.parse_args()
 
+terraform_aws_tagging_template = """
```

Comment:
It would be preferable to not generate these files.

Reply:
Changed tagging.tf to module located in `/infra`